.rubocop_todo.yml

```diff
@@ -55,7 +55,7 @@ Metrics/MethodLength:
 # Offense count: 2
 # Configuration parameters: CountComments.
 Metrics/ModuleLength:
-  Max: 176
+  Max: 200
```

Comment:
Fyi usually we just run `rubocop -a ; rubocop --auto-gen-config`.

Reply:
Ah ok, I was wondering why 176 was the old value :)